docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,36 +1,44 @@
+- Add MTLS secret generation by namespace (https://github.com/hazelcast/hazelcast-platform-operator/pull/585[#585])
+- Remove `Security Context Constraints` from Operator and Management Center Permissions. The operator won't have support for hostPath persistence in OpenShift environments (https://github.com/hazelcast/hazelcast-platform-operator/pull/581[#581])
+- Add connectivity check to WAN replication CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/541[#541])
+- Change `podManagementPolicy` to `Parallel` to improve cluster setup time (https://github.com/hazelcast/hazelcast-platform-operator/pull/539[#539])
+- Support for the cluster-scoped operator is improved in the phone home package and Kubebuilder roles (https://github.com/hazelcast/hazelcast-platform-operator/pull/521[#521])
+- Update Hazelcast Go client to 1.4.0 (https://github.com/hazelcast/hazelcast-platform-operator/pull/663[#663])
+- Update default agent version to 0.1.16 (https://github.com/hazelcast/hazelcast-platform-operator/pull/690[#690])
+- Add phone home for watched namespace type becomes available (https://github.com/hazelcast/hazelcast-platform-operator/pull/616[#616])
 
 == Fixed Issues
-
-* A HotBackup can now be created before the Hazelcast cluster is ready (https://github.com/hazelcast/hazelcast-platform-operator/pull/503[#503])
-* Added webhook support for Operator Lifecycle Management (OLM) (https://github.com/hazelcast/hazelcast-platform-operator/pull/540[#540])
-* Sets a Hazelcast resource message when attempting to restore a hot backup that does not exist (https://github.com/hazelcast/hazelcast-platform-operator/pull/514[#514])
-* Multiple CRs for data structures can now be created and then stored correctly in the ConfigMap (https://github.com/hazelcast/hazelcast-platform-operator/pull/501[#501])
-* Added a security context to all containers for hostPath permissions (https://github.com/hazelcast/hazelcast-platform-operator/pull/485[#485])
-* Fixed the incorrect assignment of a variable in WanRep ConfigMap. After a restart, cluster members will now continue replicating map entries (https://github.com/hazelcast/hazelcast-platform-operator/pull/554[#554])
-* Hazelcast member status no longer contains an empty pod name field (https://github.com/hazelcast/hazelcast-platform-operator/pull/456[#456])
+- Default WAN Replication is available through Hazelcast service on port 5710, and if no serviceType is specified during configuration, it will be set to LoadBalancer (https://github.com/hazelcast/hazelcast-platform-operator/pull/706[#706])
+- Make Hazelcast container ports dynamically updatable and remove infinite triggers (https://github.com/hazelcast/hazelcast-platform-operator/pull/692[#692])
+- Fix `CreateOrUpdate` function usage to prevent Map CR from being deleted accidentally (https://github.com/hazelcast/hazelcast-platform-operator/pull/686[#686])
```

Comment:
```suggestion
- Fixed `CreateOrUpdate` function usage to prevent the accidental deletion of the Map CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/686[#686])
```